airbyte-integrations/connectors/source-instatus/source_instatus/instatus.yaml

```diff
@@ -0,0 +1,286 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: [ ]
+
+  user_field_selector:
+    extractor:
+      field_pointer: [ "user" ]
+
+  requester:
+    url_base: "https://api.instatus.com/"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_key'] }}"
+
+  public_data_requester:
+    url_base: ""
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_key'] }}"
+    error_handler:
+      type: CompositeErrorHandler
+      error_handlers:
+        - type: DefaultErrorHandler
+          response_filters:
+            # 401 This user is not an owner of the status page
+            - http_codes: [ 401 ]
+              action: IGNORE
+        - response_filters:
+            - http_codes: [ 429 ]
+              action: RETRY
+          backoff_strategies:
+            - type: "ConstantBackoffStrategy"
+              backoff_time_in_seconds: 310
```

Comment:
Does API return the `RetryAfter` header for 429?

Reply:
Yes, thank you, I replaced `ConstantBackoffStrategy` with `WaitTimeFromHeader`.